client/apis/volumesnapshot/v1/types.go

```diff
@@ -332,6 +332,7 @@ type VolumeSnapshotContentSpec struct {
 	// SourceVolumeMode is the mode of the volume whose snapshot is taken.
 	// Can be either “Filesystem” or “Block”.
 	// If not specified, it indicates the source volume's mode is unknown.
+	// This field is immutable.
```

Comment:
I think we should also add a note that this is an alpha field.

Reply:
Done!